Tests/SwiftParserTest/StatementTests.swift

```diff
@@ -96,7 +96,7 @@ final class StatementTests: XCTestCase {
   }
 
   func testNestedIfs() {
-    let nest = 20
+    let nest = 10
```

Comment:
What was this one for? Do we hit the stack limit in debug at only 10 😓?

Reply:
I honestly don’t know what this test was meant to test. But I figured that there won’t really be any issues that we’d catch with 20 nested `if` statements that we wouldn’t catch if we nest 10.

And yes, this was hitting the maximum nesting level in debug builds. Not entirely sure why.